text/0000-raw-txn-header.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: raw_txn_header
+- Start Date: 2018-08-03
+- RFC PR:
+- Sawtooth Issue:
+
+# Summary
+[summary]: #summary
+
+Requesting to add API to the sdk - txn.get_raw_header(), that will return
```

Comment:
Replace this sentence with something like "This RFC proposes an additional API call, txn.get_raw_header(), which will return the transaction's header bytes."